pypeit/extraction.py

```diff
@@ -761,6 +761,7 @@ def local_skysub_extract(self, global_sky, sobjs, bkg_redux_global_sky=None,
             sn_gauss = self.par['reduce']['extraction']['sn_gauss']
```

Comment:
I note this here so I don't forget (!), but there appears to be a bunch of deimos sensfuncs updated here. Is that intentional? Also, Please remember to remove the `pypeit/data/standards/PHOENIX_10000K_4p0.dat` file from the PR - thanks!

Reply:
Thank you for the reminder, done!